frontend/src/components/Pages/RobotPage/LocalizationSection.tsx

```diff
@@ -8,6 +8,39 @@ interface RobotProps {
 }
 
 export function LocalizationSection({ robot }: RobotProps) {
+    const [selectedAssetDeck, setSelectedAssetDeck] = useState<AssetDeck>()
+    const [assetDecks, setAssetDecks] = useState<AssetDeck[]>()
+
+    useEffect(() => {
+        BackendAPICaller.getAssetDecks().then((response: AssetDeck[]) => {
+            setAssetDecks(response)
+        })
+    }, [])
```

Comment:
this will only run once. Is that intentional?

Reply:
This was with an assumption that assetDeck is something that rarely/never change creating a limited need to update